### PR TITLE
Update from RC2 to RTM

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Review by wadepickett — APPROVED** [review]
Looks good.  Thanks for the include update too.

**Comment by wadepickett** [comment]
Just a heads up, this marked back to a draft after requesting a review.  If more changes are added at this point I will watch for a review re-request.

File: aspnetcore/blazor/fundamentals/routing.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ Slashes and segments of the captured path are decoded. For a route template of `
 
 ::: moniker range="< aspnetcore-5.0"
 
-Catch-all route parameters are supported in ASP.NET Core in .NET 5 Release Candidate 1 (RC1) or later.*
+Catch-all route parameters are supported in ASP.NET Core in .NET 5 or later.*
```
Comment (scottaddie) [inline aspnetcore/blazor/fundamentals/routing.md:209]:
```suggestion
Catch-all route parameters are supported in ASP.NET Core 5.0 or later.
```

**Review by scottaddie — APPROVED** [review]
One minor suggestion to consider, and then this looks good.

File: aspnetcore/data/ef-mvc/intro.md [hunk 2]
```diff
@@ -96,8 +96,8 @@ The EF SQL Server package and its dependencies, `Microsoft.EntityFrameworkCore`
 Add the [Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore) NuGet package and the [Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore) NuGet package. In the Program Manager Console (PMC), enter the following commands to add the NuGet packages:
 
 ```powershell
-Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore -Version 5.0.0-rc.2.20475.17
-Install-Package Microsoft.EntityFrameworkCore.SqlServer -Version 5.0.0-rc.2.20475.6
+Install-Package Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
+Install-Package Microsoft.EntityFrameworkCore.SqlServer -Version
```
Comment (scottaddie) [inline aspnetcore/data/ef-mvc/intro.md:100]:
```suggestion
Install-Package Microsoft.EntityFrameworkCore.SqlServer
```

**Comment by Rick-Anderson** [comment]
@scottaddie good catch. Can you merge this tomorrow when the coast is clear then merge to live?

**Comment by scottaddie** [comment]
@Rick-Anderson Yes, I'll add this to my checklist for tomorrow morning.

**Review by scottaddie — APPROVED** [review]

**Pull request merged**